components/chef-ui-library/src/app.tsx

```diff
@@ -2,10 +2,10 @@
 // cause side effects we need to disable that rule in this case.
 /* tslint:disable:no-import-side-effect */
 import '@stencil/router';
+import '@stencil/redux';
 /* tslint:enable:no-import-side-effect */
 
-import { Component, Prop } from '@stencil/core';
-import { Store } from '@stencil/redux';
+import { Component, Prop, h } from '@stencil/core';
```

Comment:
What is this mysterious `h` I see everywhere?

Reply:
https://github.com/ionic-team/stencil/blob/master/BREAKING_CHANGES.md#import--h--is-required